src/tools/illink/test/Mono.Linker.Tests.Cases/DataFlow/MethodParametersDataFlow.cs

```diff
@@ -260,10 +262,152 @@ class UnsupportedType ()
 			}
 
 			[UnexpectedWarning ("IL2072", Tool.Analyzer, "https://github.com/dotnet/runtime/issues/101211")]
-			public static void Test () {
+			static void TestUnsupportedType ()
+			{
 				var t = GetUnsupportedTypeInstance ();
 				RequirePublicMethods (t);
 			}
+
+			static Type[] GetTypeArray () => null;
+
+			[ExpectedWarning ("IL2098")]
+			[UnexpectedWarning ("IL2067", Tool.Analyzer, "https://github.com/dotnet/runtime/issues/101211")]
+			static void RequirePublicMethods (
+				[DynamicallyAccessedMembers (DynamicallyAccessedMemberTypes.PublicMethods)]
+				Type[] types)
+			{
+				RequirePublicFields (types);
+			}
+
+			[ExpectedWarning ("IL2098")]
+			static void RequirePublicFields (
+				[DynamicallyAccessedMembers (DynamicallyAccessedMemberTypes.PublicFields)]
+				Type[] types)
+			{
+			}
+
+			[UnexpectedWarning ("IL2072", Tool.Analyzer, "https://github.com/dotnet/runtime/issues/101211")]
+			static void TestTypeArray ()
+			{
+				var types = GetTypeArray ();
+				RequirePublicMethods (types);
+			}
+
+			static unsafe Type* GetTypePtr () => throw null;
+
+			[ExpectedWarning ("IL2098")]
+			[UnexpectedWarning ("IL2067", Tool.Analyzer, "https://github.com/dotnet/runtime/issues/101211")]
+			static unsafe void RequirePublicMethods (
+				[DynamicallyAccessedMembers (DynamicallyAccessedMemberTypes.PublicMethods)]
+				Type* typePtr)
```

Comment:
Just to make sure I understand what's going on here: we do not consider pointers of `Type` to be interesting, right? Or is #101211 saying that we shouldn't be doing that, but we are?